src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -12,60 +13,63 @@
 
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.team4909.robot.subsystems.elevator.commands.ElevatorOperatorControl;
 import frc.team4909.robot.subsystems.elevator.commands.SetElevatorPosition;
 import frc.team4909.robot.Robot;
 import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.RobotMap;
 import frc.team4909.robot.operator.controllers.BionicF310;
 
 public class ElevatorSubsystem extends Subsystem{
-    WPI_VictorSPX VictorSPX1, VictorSPX2, VictorSPX3;
-    WPI_TalonSRX masterSRX;
+    WPI_VictorSPX leftSPX, rightSPX1, rightSPX2;
+    WPI_TalonSRX leftSRX;
 
     public int holdingPosition;
 
-    public void ElevatorSubsytem(){
+    public ElevatorSubsystem() {
 
         //Lift
-        masterSRX = new WPI_TalonSRX(RobotConstants.elevatorSRXID);  //master SRX
-        VictorSPX1 = new WPI_VictorSPX(RobotConstants.elevatorSPX1ID); //slave SPX 1
-        VictorSPX2 = new WPI_VictorSPX(RobotConstants.elevatorSPX2ID); //slave SPX 2
-        VictorSPX3 = new WPI_VictorSPX(RobotConstants.elevatorSPX3ID); //slave SPX 3
+        leftSRX = new WPI_TalonSRX(RobotMap.elevatorSRXID);  //master SRX
+        leftSPX = new WPI_VictorSPX(RobotMap.elevatorSPX1ID); //slave SPX 1
+        rightSPX1 = new WPI_VictorSPX(RobotMap.elevatorSPX2ID); //slave SPX 2
+        rightSPX2 = new WPI_VictorSPX(RobotMap.elevatorSPX3ID); //slave SPX 3
 
-        VictorSPX1.follow(masterSRX);  //All slaves will output the same value as the master SRX
-        VictorSPX3.follow(masterSRX);  
-        VictorSPX2.follow(masterSRX);
-        masterSRX.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative);
+        
 
-    }
-
-    @Override
-    public void periodic() {
-        //Sets speed to manipulator gamepad right Y stick value
-        double moveSpeed = Robot.manipulatorGamepad.getThresholdAxis(BionicF310.RY) * RobotConstants.elevatorSpeedMultiplier;
-
-        if(moveSpeed == 0 ) {  //If Y-stick value is not moving, HOLD position
-            masterSRX.set(ControlMode.Position, holdingPosition);
-        } 
-        else{ //Set speed to Y-stick value and HOLD position
-            masterSRX.set(ControlMode.PercentOutput, moveSpeed);
-            holdCurrentPosition();
-        }
-    }
+        leftSPX.follow(leftSRX);  //All slaves will output the same value as the master SRX
+        rightSPX1.follow(leftSRX);  
+        rightSPX2.follow(leftSRX);
+        
+        rightSPX1.setInverted(true);
+        rightSPX2.setInverted(true);
+        leftSRX.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative);
+        leftSRX.selectProfileSlot(1, 0);
```

Comment:
i would recommend just using slot 0 for all of the config, including the below PID constants